eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/parameters/InputParameters.java

```diff
@@ -86,6 +99,7 @@ public JobParameters toJobParameters() {
         return new JobParametersBuilder()
                 .addString("assemblyAccession", assemblyAccession)
                 .addString("projectAccession", projectAccession)
+                .addString("projects", CollectionUtils.isEmpty(projects) ? "" : String.join(",", projects))
```

Comment:
We should never have to use this reader for them but some of the dbSNP study name contains commas (,) which would screw this up.

Reply:
This reader can't even read from the dbSNP collection... but yes, I'll add a check just in case we do something dumb.